client/src/components/ExperienceRow.js

```diff
@@ -53,7 +53,7 @@ const ExperienceRow = ({
         {languages.map((lang) => {
           return (
             <MenuItem key={lang} value={lang} className={classes.dropDownText}>
-              {lang}
+              {lang === "cplusplus" ? "c++" : lang}
```

Comment:
Let's add a ticket to change the enum in the back-end to be something like this:

```
class Languages(object):
    LANGUAGES = {
             "c++": "c++",
             "javascript": "javascript",
...
}
```

Language.LANGUAGES[lang]
if lang in Language.LANGUAGES